<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView_day">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_day">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить строку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_day">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить строку</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_net">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>531</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>day table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>net table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_net">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить строку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_net">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить строку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hours">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Определить кол-во подключений за:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hours">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hours_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Определить объём полученной информации за период:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_period_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ПО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_period_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_period_result">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sites">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Определить посещенные сайты:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sites">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>часов:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
